File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/HarvestsUpdateHarvestRenameRequestItem.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class HarvestsUpdateHarvestRenameRequestItem {
    @JsonProperty("Id")
    public Integer id;
    @JsonProperty("NewName")
    public String newName;
    @JsonProperty("OldName")
    public String oldName;
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/HarvestsUpdateHarvestRenameRequestItem.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class HarvestsUpdateHarvestRenameRequestItem(
    @JsonProperty("id")
    val id: Int? = null,
    @JsonProperty("newName")
    val newName: String? = null,
    @JsonProperty("oldName")
    val oldName: String? = null
)
